docs/api.md

```diff
@@ -3,103 +3,776 @@ layout: default
 title: API Reference 
```

Comment:
Re-gen the docs after spelling fix, helm-controller is still misspelled

Reply:
Fixed